content/neuromorphic-computing/software/data-tools/neuromorphic-drivers/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Neuromorphic drivers"
+type: neuromorphic-software
+description: "Python and Rust libraries to interact with event cameras in real-time."
+logo: "neuromorphic-drivers.png"
+website: "https://github.com/neuromorphicsystems/neuromorphic-drivers"
+dependencies: "NumPy, Rust"
+field_of_application: "Drivers"
+source_code: "https://github.com/neuromorphicsystems/neuromorphic-drivers"
+pypi_id: "neuromorphic-drivers"
+license: "MIT"
+supports_hardware: false
+supports_NIR: false
+language: "Python, Rust"
+maintainer:
+  - "Alexandre Marcireau"
+draft: false
+---
+
+## Overview
+
+Neuromorphic drivers is a library to interact with USB event cameras in real-time. It is compatible with all major operating systems (Linux x64 and ARM, macOS x64 and ARM, and Windows x64) and it aims to support as many commercial devices as possible.
+
+The library can be used in Python and Rust.
+
+By design, Neuromorphic drivers provides no processing algorithms. It may instead be combined with other libraries (for instance https://open-neuromorphic.org/neuromorphic-computing/software/data-tools/tonic/ or https://open-neuromorphic.org/neuromorphic-computing/software/data-tools/faery/) to build real-time processing pipelines.
```

Comment:
```By design, Neuromorphic drivers provides no processing algorithms. It may instead be combined with other libraries (for instance [Tonic](/neuromorphic-computing/software/data-tools/tonic/) or [Faery](/neuromorphic-computing/software/data-tools/faery/) to build real-time processing pipelines.```

I'm sorry, I meant the bit in the code block which is a bit more refined:

By design, Neuromorphic drivers provides no processing algorithms. It may instead be combined with other libraries (for instance [Tonic](/neuromorphic-computing/software/data-tools/tonic/) or [Faery](/neuromorphic-computing/software/data-tools/faery/) to build real-time processing pipelines.